iOS/Flows/Home/_New/new_MessageCell.swift

```diff
@@ -10,16 +10,30 @@ import Foundation
 
 class new_MessageCell: UICollectionViewCell {
 
-    let textView = TextView()
-    let replyCountLabel = Label(font: .regular)
+    private let messageContainerView = UIView()
+    private let messageTextView = TextView()
+
+    private let repliesContainerView = UIView()
+    private let replyCountLabel = Label(font: .regular)
+    private let repliesTextView = TextView()
```

Comment:
Created some rounded containers to hold the main message and the latest threaded message.